File: TeamFlow-FrontEnd/src/app/core/models/app-roles.ts
export const AppRoles = {
  Admin: 'Admin',
  TeamLeader: 'TeamLeader',
  Developer: 'Developer',
  QA: 'QA'
} as const;

export type AppRole = typeof AppRoles[keyof typeof AppRoles];

export const APP_ROLES: AppRole[] = [
  AppRoles.Admin,
  AppRoles.TeamLeader,
  AppRoles.Developer,
  AppRoles.QA
];

File: TeamFlow/TeamFlow/Authorization/AppRoles.cs
namespace TeamFlow.Authorization
{
    public static class AppRoles
    {
        public const string Admin = "Admin";
        public const string TeamLeader = "TeamLeader";
        public const string Developer = "Developer";
        public const string QA = "QA";

        public static readonly ISet<string> All = new HashSet<string>(StringComparer.Ordinal)
        {
            Admin,
            TeamLeader,
            Developer,
            QA
        };
    }
}
